python-package/xgboost/sklearn.py

```diff
@@ -517,7 +517,7 @@ def feature_importances_(self):
             raise AttributeError('Feature importance is not defined for Booster type {}'
                                  .format(self.booster))
         b = self.get_booster()
-        fs = b.get_fscore()
+        fs = b.get_score(importance_type='gain')
```

Comment:
I think that there could be `importance_type` parameter in the constructor for letting users decide what type of importance they need, like it has been recently [done in LightGBM](https://github.com/Microsoft/LightGBM/pull/1470).